python/orion_finance_sdk_py/cli.py

```
@@ -193,8 +143,206 @@ def update_fee_model(
+                _update_fee_model_logic(
+                    fee_type_to_int[fee_type_str],
+                    int(perf_fee * BASIS_POINTS_FACTOR),
+                    int(mgmt_fee * BASIS_POINTS_FACTOR),
+                )
+
+            input("\nPress Enter to continue...")
+
+        except KeyboardInterrupt:
+            print("\nOperation cancelled.")
+            continue  # Go back to main menu loop
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)